.gitignore

```diff
@@ -15,13 +16,22 @@ config*.log
 /native/**/*.a
 /native/**/*.so
 /native/**/*.vcxproj.user
+/native/**/*.aps
 /native/**/Win32/
 /native/**/x64/
+/native/**/Release
+/native/exe/HCWIRBlasterCOM/HCWIRBlasterCOM_*
 
+/third_party/ffmpeg/lib*/*_list.c
+/third_party/ffmpeg/lib*/*.ver 
+/third_party/ffmpeg/doc/*.3
+/third_party/ffmpeg/*/*.a
+/third_party/ffmpeg/*/*.def
 /third_party/**/*.o
 /third_party/**/*.lo
 /third_party/**/*.la
 /third_party/**/*.pc
+/third_party/**/*.dll
```

Comment:
This one also needs to be modified, there's .dll files in the following third_party directories:

mingw
technotrend
JOGL
JDIC

Reply:
Will do.